src/vm/classnames.h

```diff
@@ -27,6 +27,7 @@
 #ifdef FEATURE_COMINTEROP
 #define g_ECMAKeyToken "B77A5C561934E089"       // The ECMA key used by some framework assemblies: mscorlib, system, etc.
 #define g_FXKeyToken "b03f5f7f11d50a3a"         // The FX key used by other framework assemblies: System.Web, System.Drawing, etc.
+#define g_MscorlibAsmName "System.Private.CoreLib"
```

Comment:
I would call it just `g_CorelibAsmName`. (It is what we have been trying to do in other places.)